takwimu/static/css/takwimu.css

```diff
@@ -0,0 +1,274 @@
+      border: 3px solid #fff;
+      border-radius: 9999px; }
+    .navbar.fixed-top .dropdown-menu-lg .country-flag-links div a:hover {
+      color: #fff; }
+
+/*
+* Profile content css
+* --------------------
+*/
+.profile-content {
+  /* Ideally only the section title would underline on mouse hover */ }
+  .profile-content .is-active {
+    background-color: #fafafa;
+    /* Quick way to remove underline on mouse hover for the current selected item */ }
+    .profile-content .is-active + div.border-top {
+      border-top: 0 !important; }
+    .profile-content .is-active h1.section-title:hover {
+      text-decoration: none; }
+  .profile-content h1.section-title:hover, .profile-content h1.section-title:focus {
+    text-decoration: underline; }
+
+/*
+* Country container css
+* ----------------------
+*/
+.country-flag-links .col {
+  padding: 3em 1em;
+  border-right: 1px solid #ddd; }
+  .country-flag-links .col a {
+    text-decoration: none;
+    color: #000000; }
+    .country-flag-links .col a svg {
+      display: block;
+      margin: 0 auto;
+      width: 50px;
+      height: 50px; }
+    .country-flag-links .col a span {
+      padding-top: 1.5em;
+      display: block;
+      font-weight: 400; }
+    .country-flag-links .col a:hover, .country-flag-links .col a:focus {
+      color: #1e4a7e; }
+      .country-flag-links .col a:hover svg, .country-flag-links .col a:focus svg {
+        filter: #1e4a7e;
+        /* IE6-9 */
+        -webkit-filter: grayscale(1);
+        /* Google Chrome, Safari 6+ & Opera 15+ */
+        filter: grayscale(1);
+        /* Microsoft Edge and Firefox 35+ */ }
+    .country-flag-links .col a.active {
+      color: #6c757d; }
+      .country-flag-links .col a.active svg {
+        filter: #6c757d;
+        /* IE6-9 */
+        -webkit-filter: grayscale(1);
+        /* Google Chrome, Safari 6+ & Opera 15+ */
+        filter: grayscale(1);
+        /* Microsoft Edge and Firefox 35+ */ }
+    .country-flag-links .col a[data-toggle="tooltip"] * {
+      opacity: 0.5; }
+
+/*
+* Extra small devices (portrait phones, less than 576px)
+* Small devices (landscape phones, 576px and down)
+*/
+@media (max-width: 576px) {
+  .nav-item {
+    font-size: 0.75em; }
+  .country-container div.col:nth-child(even) {
+    border-right: 1px solid transparent; } }
+
+/*
+* Small devices (landscape phones, 576px and up)
+* Medium devices (tablets, 768px and up)
+*/
+@media (min-width: 576px) and (max-width: 768px) {
+  /*
+  * Do not delete the code below, it's useful for css debuging
+  * during development
+  */ }
+
+/*
+* Large devices (desktops, 992px and up)
+*/
+@media (min-width: 768px) and (max-width: 992px) {
+  /*
+  * Do not delete the code below, it's useful for css debuging
+  * during development
+  */ }
+
+/*
+* Extra large devices (large desktops, 1200px and up)
+*/
+@media (min-width: 1200px) {
+  /*
+  * Do not delete the code below, it's useful for css debuging
+  * during development
+  */ }
```

Comment:
@phillipahereza Are you using docker-compose? My setup doesn't create this file.. 🤔

Reply:
@DavidLemayian sorry, I meant to remove that after populating the DB